doc/ikfast/ikfast_tutorial.rst

```diff
@@ -73,210 +52,83 @@ Inside your catkin workspace's ``./src`` directory: ::
-
- cd ~/catkin_ws/src
- export MOVEIT_IK_PLUGIN_PKG="$MYROBOT_NAME"_ikfast_"$PLANNING_GROUP"_plugin
- catkin_create_pkg "$MOVEIT_IK_PLUGIN_PKG"
-
-Build your workspace so the new package is detected (can be 'roscd'): ::
+Usage
+-----
+The IKFast plugin can be used as a drop-in replacement for the default KDL IK Solver, but with greatly increased performance. The MoveIt configuration file should be automatically edited by the generator script but in some cases this might fail. In this situation you can switch between the KDL and IKFast solvers using the *kinematics_solver* parameter in the robot's kinematics.yaml file: ::
 
- catkin build
+  rosed "$MYROBOT_NAME"_moveit_config kinematics.yaml
 
-Create the plugin source code. Run ``rosrun moveit_kinematics create_ikfast_moveit_plugin.py -h`` to see optional arguements: ::
+Edit these parts: ::
 
- rosrun moveit_kinematics create_ikfast_moveit_plugin.py "$MYROBOT_NAME" "$PLANNING_GROUP" "$MOVEIT_IK_PLUGIN_PKG" "$BASE_LINK_NAME" "$EEF_LINK_NAME" "$IKFAST_OUTPUT_PATH"
+ <planning_group>:
+   kinematics_solver: <myrobot_name>_<planning_group>_ikfast_plugin/IKFastKinematicsPlugin
 
-For the Panda we need to use some of the optional parameters: ::
+Test the Plugin
+^^^^^^^^^^^^^^^
+Use the MoveIt RViz Motion Planning Plugin and use the interactive markers to see if correct IK Solutions are found.
 
- rosrun moveit_kinematics create_ikfast_moveit_plugin.py --moveit_config_pkg="panda_moveit_config" --robot_name_in_srdf="panda" "$MYROBOT_NAME" "$PLANNING_GROUP" "$MOVEIT_IK_PLUGIN_PKG" "$BASE_LINK_NAME" "$EEF_LINK_NAME" "$IKFAST_OUTPUT_PATH"
+Updating the Plugin
+-------------------
 
+If any future changes occur with MoveIt! or IKFast, you might need to re-generate this plugin using our scripts. To facilitate this, a bash script was automatically created in the root of your IKFast MoveIt package, named *update_ikfast_plugin.sh*. This regenerates the plugin from the OpenRAVE-generated .cpp solver file.
 
-Or without ROS: ::
+Tweaking the creation process
+-----------------------------
 
- python /path/to/create_ikfast_moveit_plugin.py "$MYROBOT_NAME" "$PLANNING_GROUP" "$MOVEIT_IK_PLUGIN_PKG" "$BASE_LINK_NAME" "$EEF_LINK_NAME" "$IKFAST_OUTPUT_PATH"
+The process of creating the IKFast MoveIt plugin comprises several steps:
 
-Usage
------
-The IKFast plugin should function identically to the default KDL IK Solver, but with greatly increased performance. The MoveIt configuration file is automatically edited by the moveit_ikfast script but you can switch between the KDL and IKFast solvers using the *kinematics_solver* parameter in the robot's kinematics.yaml file: ::
+1. Download (and slightly modify) the docker image provided by `personalrobotics <https://hub.docker.com/r/personalrobotics/ros-openrave>`_
+2. Convert the ROS URDF file to Collada required for OpenRAVE: ::
 
- rosed "$MYROBOT_NAME"_moveit_config kinematics.yaml
+     rosrun collada_urdf urdf_to_collada <myrobot_name>.urdf <myrobot_name>.dae
```

Comment:
use $MYROBOT_NAME

Reply:
Yep. This one I forgot to change. Fixed.